n8n/helm/n8n/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: n8n
 description: helm chart for n8n
 type: application
 version: 0.1.6
-appVersion: "0.202.1"
+appVersion: "0.212.1"
 dependencies:
-- name: postgres
```

Comment:
It's not very important, but correct yaml formatting actually doesn't have an indent for array items.

Reply:
done